src/main/java/servlet/ServletStart.java

```diff
@@ -0,0 +1,18 @@
+package servlet;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+public class ServletStart extends HttpServlet {
+
+    public static boolean isInitialized=false;
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        isInitialized=false;

```

Comment:
What is this and why do you use it everywhere?

Reply:
it checks whether a transition to the next page was made in order to avoid errors in the code algorithm

Reply:
Not understand.
Let's discuss on the lesson